apps/rush-lib/src/data/EnvironmentConfiguration.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+const ENV_VALUE_PREFIX: string = 'rush_';
+
+export enum EnvironmentValue {
+  TempDirectoryOverride = 'rush_tempDir'
```

Comment:
>rush_tempDir [](start = 27, length = 12)

Environment variables should always be ALL_CAPS_WITH_UNDERSCORES.  This is the most portable convention.  It is standardized for certain shells such as Bash, because their environment variables are case-sensitive.